### DISCUSSION TITLE
【部品購入希望】「平頭ネジ M1.7」2本、「小頭ネジ M1.7」2本を紛失しました。

### DISCUSSION THREAD

**Discussion by naitokosuke** [Yowkees/keyball#267] [Q&A] [answered]
タイトルの通り、以下の2部品について紛失してしまいました。
- 「平頭ネジ M1.7」2本、
- 「小頭ネジ M1.7」2本

お手数ですが購入方法を教えていただきたいです。

また、こちらの部品についてですが、キットに同封されている紙とGitHub上とで表記揺れを確認しましたので併せてご確認いただけますと幸いです。
- 紙: 「平頭ネジ M1.7」、「小頭ネジ M1.7」
- GitHub 「平頭**タップ**ネジ M1.7」、「小頭**タップ**ネジ M1.7」
![FD93D56E-A106-4FBC-A719-E350A4B690B6_1_102_o](https://github.com/Yowkees/keyball/assets/102337893/0ee18d31-562b-433f-afa5-837b1355d64a)

以上よろしくお願いします。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。了解しました。

ねじ単体での販売可能です。配送先等をお話ししたいので、下記リンクの白銀ラボウェブショップのお問い合わせよりご連絡いただけますでしょうか。
https://shirogane-lab.com/pages/contact

お手数ですがよろしくお願いいたします。
